docs/public/deploying-a-devfile-using-odo.adoc

```diff
@@ -28,6 +28,8 @@ An example deployment scenario:
  .. Synchronizes your source code to the containers
  .. Executes any prerequisite commands
 
+*Note that odo's devfile support is currently in preview, and requires enabling experimental mode*
```

Comment:
No need to add this, we already have a disclaimer at the beginning (see pre-requisites) saying that experimental mode is required.

Reply:
See line: https://github.com/openshift/odo/blame/e6b69bc46888a417eb6ea0b912d6e4494683c7c0/docs/public/deploying-a-devfile-using-odo.adoc#L39

Reply:
Thanks, I somehow missed that haha 😅